docs/user/cli.md

```diff
@@ -54,7 +54,7 @@ autorest --input-file=myfile.json --output-folder=./generated/code/ --namespace=
 |`--input-file=FILENAME`|Adds the given file to the list of input files for generation process|
 |`--output-folder=DIRECTORY`|The location for generated files. If not specified, uses `./Generated` as the default|
 |`--namespace=NAMESPACE`|sets the namespace to use for the generated code|
-|`--license-header=HEADER`| Text to include as a header comment in generated files. Use NONE to suppress the default header.|
+|`--license-header|licence-header=HEADER`| Text to include as a header comment in generated files. Use NONE to suppress the default header.|
```

Comment:
TBH, I'd not document this. Since it's just an internal redirect (as we may do more in the future), I think documentation should be as crisp and brief as possible, even if that means dropping some alternative ways of achieving one and the same :)